Templates/CSharp/EdgeDriverTemplate/source.extension.vsixmanifest

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="EdgeDriverTest.0b43ac6b-e6ff-4afc-a5f6-286d41c046a6" Version="16.0.28113.00" Language="en-US" Publisher="Microsoft" />
+        <DisplayName>Web Driver Test Template For Edge</DisplayName>
+        <Description xml:space="preserve">Create an automatic UI test using Edge Driver.</Description>
+        <PackageId>Microsoft.VisualStudio.Templates.CS.MSTestv2.EdgeDriverTest</PackageId>
+    </Metadata>
+    <Installation AllUsers="true" SystemComponent="true">
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0]" />
+        <InstallationTarget Version="[16.0]" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[16.0]" Id="Microsoft.VisualStudio.Enterprise" />
+        <InstallationTarget Version="[16.0]" Id="Microsoft.VisualStudio.VSWinExpress" />
+        <InstallationTarget Version="[16.0]" Id="Microsoft.VisualStudio.VWDExpress" />
+        <InstallationTarget Version="[16.0]" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
+    </Installation>
+    <Dependencies>
+        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
```

Comment:
Why does vstemplate file list 4.6 as minimum required version while the vsixmanifest depends on 4.5 as minimum version.

Reply:
should be 4.5